.github/workflows/update-labels.yml

```diff
@@ -0,0 +1,18 @@
+name: Update label when user responds
+permissions:
+  issues: write
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  run-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove label when user responds
+        uses: andymckay/labeler@master
+        if: ${{ github.event.comment.user.login == github.event.issue.user.login && contains(github.event.issue.labels.*.name, 'wait for update') && !contains(github.event.issue.labels.*.name, 'user responded') }}
```

Comment:
What happends if the user respond an issue with the label "no response"?
Should we change the label "no response" to "user responded"?

Reply:
Hmm let me implement that.